checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -151,37 +195,54 @@ def analyze_line(
     ) -> set[PotentialSecret]:
         is_iac = f".{filename.split('.')[-1]}" not in SOURCE_CODE_EXTENSION
         filetype = determine_file_type(filename)
-        multiline_parser = MULTILINE_PARSERS.get(filetype)
+        multiline_parsers = MULTILINE_PARSERS.get(filetype)
 
         if len(line) <= MAX_LINE_LENGTH:
             if is_iac:
                 # classic key-value pair
                 keyword_on_key = self.keyword_scanner.analyze_line(filename, line, line_number, **kwargs)
                 if keyword_on_key:
-                    return self.detect_secret(
+                    potential_secrets = self.detect_secret(
                         scanners=self.high_entropy_scanners_iac,
                         filename=filename,
                         line=line,
                         line_number=line_number,
                         kwargs=kwargs
                     )
+                    if potential_secrets and ignore_secret(
+                        file_type=filetype,
+                        context=context,
+                        raw_context=raw_context,
+                    ):
+                        # to reduce false-positives we check the context of the finding
+                        return set()
+
+                    return potential_secrets
 
                 # not so classic key-value pair, from multiline, that is only in an array format.
                 # The scan is one-way backwards, so no duplicates expected.
```

Comment:
my mistake- but can we change the comment here to something like this:
The scan is both forward and backword, searching for a pair for a potential secret.

Reply:
sure 😄